### PR TITLE
3.0 preview dev

### PR DESCRIPTION
**Pull request by notwaldorf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,88 @@
...
-## Polymer.IronValidatableBehavior
-
-`Use Polymer.IronValidatableBehavior` to implement an element that validates user input.
-Use the related `Polymer.IronValidatorBehavior` to add custom validation logic to an iron-input.
-
-By default, an `<iron-form>` element validates its fields when the user presses the submit button.
-To validate a form imperatively, call the form's `validate()` method, which in turn will
-call `validate()` on all its children. By using `Polymer.IronValidatableBehavior`, your
-custom element will get a public `validate()`, which
+## &lt;iron-validatable-behavior&gt;
```
Comment (bicknellr) [inline README.md]: `&lt;iron-validatable-behavior&gt;` -> `` `IronValidatableBehavior` ``

File: README.md [hunk 2]
```diff
@@ -1,46 +1,88 @@
...
+    }
+  }
+}
+customElements.define('sample-element', SampleElement);
+```
+
+### In an html file using the element
+```html
+<html>
+  <head>''
```
Comment (bicknellr) [inline README.md]: There's some extra quotes here.

**Review by bicknellr — APPROVED** [review]
LGTM; a couple of comments.

**Review by bicknellr — CHANGES_REQUESTED** [review]
There's a few places in here where `Polymer.IronValidatableBehavior` should be changed to `IronValidatableBehavior`.

File: README.md (hunk 1 above)
Comment (notwaldorf) [reply README.md]: ha my bad

**Review by bicknellr — APPROVED** [review]

**Pull request merged**